packages/faceted-search/src/components/Badges/BadgeCheckboxes/BadgeCheckboxesForm.component.js

```diff
@@ -51,15 +52,11 @@ const getCheckboxes = (checkboxes, value, filterValue) => {
 	const formatFilterValue = filterValue.trim().toLocaleLowerCase();
 
 	return checkboxes
-		.filter(checkbox =>
-			get(checkbox, 'label', '')
-				.toLocaleLowerCase()
-				.includes(formatFilterValue),
-		)
+		.filter(checkbox => get(checkbox, 'label', '').toLocaleLowerCase().includes(formatFilterValue))
 		.map(createCheckboxEntity(value));
 };
 
-const BadgeCheckboxesForm = ({ checkboxValues, id, onChange, onSubmit, value, t }) => {
+const BadgeCheckboxesForm = ({ checkboxValues, id, onChange, onSubmit, value, label, t }) => {
```

Comment:
`label` seems confusing since it's not a displayed and translated label. It conflicts in my head with the button label for example. It seems to be only used to get the data-feature, perhaps something like `dataFeatureKey`?

Reply:
or just "feature"

Reply:
Understand, it can be confusing, just fix